### PR TITLE
update contributors

### PR DESCRIPTION
**Pull request by weissi**

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -60,27 +68,30 @@ needs to be listed here.
 - John Connolly <connoljo2@gmail.com>
 - John Holdsworth <github@johnholdsworth.com>
 - John Lin <johnlinvc@gmail.com>
+- Joshua Rutkowski <54906829+joshrutkowski@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:71]: @joshrutkowski if you'd like a proper email address here, please let me know which.

File: CONTRIBUTORS.txt [hunk 2]
```diff
@@ -60,27 +68,30 @@ needs to be listed here.
...
 - Ludovic Dewailly <ldewailly@apple.com>
+- Luo Xiu <luoxiustm@gmail.com>
 - Marcus Liotta <marcus@liotta.se>
 - Mario Sangiorgio <msangiorgio@apple.com>
 - Markus Jais <markusjais@googlemail.com>
 - Markus Kieselmann <markus@kieselmann.eu>
 - Marli Oshlack <marli.oshlack@apple.com>
 - Matt Eaton <agnosticdev@gmail.com>
-- Max Desiatov <max.desiatov@gmail.com>
+- Max Desiatov <max@desiatov.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:94]: @MaxDesiatov I'm assuming the `max@desiatov.com` is your preferred one?

**Review by argon — APPROVED** [review]

File: CONTRIBUTORS.txt [hunk 3]
```diff
@@ -97,6 +108,7 @@ needs to be listed here.
 - Richard Low <ral@apple.com>
 - Romain Pouclet <rpouclet@apple.com>
 - Rukshani Athapathu <arukshani@users.noreply.github.com>
+- Rémi Doreau <32459935+ayshiff@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:111]: @ayshiff / @arukshani  if you'd like a proper email address here, please let me know.

File: CONTRIBUTORS.txt [hunk 4]
```diff
@@ -112,14 +124,18 @@ needs to be listed here.
...
 - Tom Counsell <tamc@greenonblack.com>
 - Tomer Doron <tomerd@apple.com>
 - Trevör <adtrevor@users.noreply.github.com>
 - Trevör Anne Denise <adtrevor@users.noreply.github.com>
 - Volodymyr Sapsai <vsapsai@users.noreply.github.com>
 - Will Lisac <will@lisac.org>
 - Wilson Ding <hello@wilsonding.com>
+- buttaface <butta@fastem.com>
 - fadi-botros <botros_fadi@yahoo.com>
+- jemmons <jemmons@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:137]: @jemmons if you'd like a proper email address (and/or a real name) here, please let me know which.

File: CONTRIBUTORS.txt (hunk 2 above)
Comment (MaxDesiatov) [reply CONTRIBUTORS.txt:94]: That's correct, thanks!

File: CONTRIBUTORS.txt (hunk 1 above)
Comment (joshrutkowski) [reply CONTRIBUTORS.txt:71]: jr.rtkski@gmail.com

Comment (weissi) [reply CONTRIBUTORS.txt]: thanks, updated.

File: CONTRIBUTORS.txt (hunk 3 above)
Comment (ayshiff) [reply CONTRIBUTORS.txt:111]: remi.d45@gmail.com, thanks!

Comment (weissi) [reply CONTRIBUTORS.txt]: thanks, done.

**Review by Lukasa — APPROVED** [review]

**Review by shivaylamba — APPROVED** [review]

**Pull request merged**